src/gui_gtk.c

```diff
@@ -565,6 +565,11 @@ static void set_labels(GtkLabels *glab, Labels *data)
 		gtk_combo_box_text_append_text(GTK_COMBO_BOX_TEXT(glab->activecore), format(_("Core #%i"), i));
 	gtk_combo_box_set_active(GTK_COMBO_BOX(glab->activecore), opts->selected_core);
 
+	gtk_widget_set_tooltip_text(glab->gtktab_cpu[NAME][FAMILY], _("BaseFamily[3:0]"));
```

Comment:
I am not sure that the `[3:0]` is relevant for end-user (that is why I dropped it for display family and model).

Reply:
You're right. I reminded that it is important to explain the relationship between `Base Family/Model` and `Disp. Family/Model`.